Tests/DatadogTests/Datadog/Mocks/RUMFeatureMocks.swift

```diff
@@ -26,9 +26,17 @@ extension RUMFeature {
 }
 
 extension DatadogCoreProxy {
-    func waitAndReturnRUMEventMatchers(file: StaticString = #file, line: UInt = #line) throws -> [RUMEventMatcher] {
-        return try waitAndReturnEventsData(of: RUMFeature.self)
+    func waitAndReturnRUMEventMatchers(removeApplicationLaunch: Bool = true,
+                                       file: StaticString = #file,
+                                       line: UInt = #line) throws -> [RUMEventMatcher] {
+        var matchers = try waitAndReturnEventsData(of: RUMFeature.self)
             .map { data in try RUMEventMatcher.fromJSONObjectData(data) }
+        if removeApplicationLaunch {
+            matchers = matchers.filter({ matcher in
+                (try? matcher.attribute(forKeyPath: "view.name")) != RUMOffViewEventsHandlingRule.Constants.applicationLaunchViewName
+            })
+        }
```

Comment:
The other option is to update all tests? what impact would that be?
I think such filtering should be per test instead of having it hidden in the `waitAndReturnRUMEventMatchers`, but It might be impactful..

Reply:
Yes, that's the other option, it's around 30 tests.  Very few tests happen to actually care about the launch view, which is why I put this here. I could make that parameter explicit, so it's not quite as hidden?